### ISSUE TITLE
Ensure site is accessible

### ISSUE THREAD

**Issue by mpitblado** [gbif/hp-beaty-biodiversity-museum#52]
Using the [a11y project checklist](https://www.a11yproject.com/checklist/) work to make the site more accessible. The criteria can be divided into a checklist, and major checklist items will be tracked here.

- [ ] Content
- [ ] Global code
- [ ] Keyboard
- [ ] Images
- [ ] Headings
- [ ] Lists
- [ ] Controls
- [ ] Tables
- [ ] Forms
- [ ] Media
- [ ] Video
- [ ] Audio
- [ ] Appearance
- [ ] Animation
- [ ] Color contrast
- [ ] Mobile and touch

This issue will be re-introduced on a scheduled basis (perhaps yearly) to ensure that any new features introduced adhere to this guidelines where possible.

**Comment by MortenHofft** [comment]
I had expected this issue would come at some point. And I counted on Canada being the first!
We have used a ui library that is focused on a11y, so that should be a good start. But I expect that work is still needed. I have long ago aired, that if someone have money to spend, having accessibility reviews could make sense at some point (though I do not believe we are ready for that yet).

Be aware that we are in the process of migrating the code to a new repo. And some things will be implemented a new in that process. So I might postpone some issues, but that doesn't mean they are ignored.
